lib/logto_client.dart

```diff
@@ -1,35 +1,40 @@
-import 'package:http/http.dart' as http;
 import 'package:flutter/material.dart';
+import 'package:http/http.dart' as http;
 import 'package:jose/jose.dart';
 
 import '/logto_core.dart' as logto_core;
+import '/src/exceptions/logto_auth_exceptions.dart';
 import '/src/interfaces/logto_interfaces.dart';
 import '/src/utilities/constants.dart';
+import '/src/utilities/id_token.dart';
 import '/src/utilities/pkce.dart';
+import '/src/utilities/token_storage.dart';
 import '/src/utilities/utils.dart' as utils;
-import '/src/utilities/id_token.dart';
 import '/src/utilities/webview_provider.dart';
-import '/src/utilities/token_storage.dart';
-import '/src/exceptions/logto_auth_exceptions.dart';
 
 export '/src/interfaces/logto_config.dart';
 
 // Logto SDK
 class LogtoClient {
   final LogtoConfig config;
-  final http.Client httpClient;
+  final http.Client _httpClient;
 
   late PKCE _pkce;
   late String _state;
 
-  // TODO: load from storage
-  TokenStorage _tokenStorage = TokenStorage();
+  static late TokenStorage _tokenStorage;
 
   OidcProviderConfig? _oidcConfig;
 
-  LogtoClient(this.config, this.httpClient);
+  LogtoClient(this.config, this._httpClient) {
+    if (TokenStorage.loaded == null) {
```

Comment:
I guess instead of initializing the token storage from the LogtoFlutter class, we could add an instance method to the client itself. That would lead to better architecture and better customizability for the user.